tests/dragonfly/instance.py

```diff
@@ -170,6 +170,11 @@ def stop(self, kill=False):
             else:
                 proc.terminate()
                 proc.communicate(timeout=15)
+                # if the return code is 0 it means normal termination
+                # if the return code is negative it means termination by signal
```

Comment:
I am not sure this is the behavior, as when I tested it when dragonfly crashed the returncode was < 0 when it did not crash the returncode was 0

Reply:
https://docs.python.org/3/library/subprocess.html#subprocess.CompletedProcess.returncode :man_shrugging:

Reply:
I know I saw the documentation, but you know there is documentation and there is code.. just make sure

Reply:
hmm my two cents here are that if the process processes the signal it would return 0 otherwise a negative number with the signal that terminated it. But from my understanding even if returns `<0` it doesn't always imply the process crashed. 

I will revert it to the original `<0` and when something related crashes we can take a look.